src/server/game/Spells/Auras/SpellAuraEffects.cpp

```diff
@@ -5043,11 +5043,14 @@ void AuraEffect::HandleAuraDummy(AuraApplication const* aurApp, uint8 mode, bool
                     if (caster && target->CanHaveThreatList())
                         target->AddThreat(caster, 10.0f);
                     break;
-                case 13139:                                     // net-o-matic
-                    // root to self part of (root_target->charge->root_self sequence
-                    if (caster)
-                        caster->CastSpell(caster, 13138, true, nullptr, this);
-                    break;
+
+              //  This Script is all wrong, it creates an additional Buff that bugs the characters. there shouldn't be here.
+              //  case 13139:                                     // net-o-matic
+              //      // root to self part of (root_target->charge->root_self sequence
+              //      if (caster)
+              //         caster->CastSpell(caster, 13138, true, nullptr, this);
+              //      break;
```

Comment:
If the intent is to not use it, delete it

Reply:
> If the intent is to not use it, delete it

Done